roles/openshift_logging/tasks/start_cluster.yaml

```diff
@@ -0,0 +1,107 @@
+---
+- shell: >
+    echo "{{openshift_logging_fluentd_nodeselector}}" | cut -d':' -f1
```

Comment:
Consider using the regex_replace or another jinja2/Ansible filter here rather than using shell, echo and cut